package.json

```diff
@@ -0,0 +1,25 @@
+{
+  "name": "week-03-tdd-strict-equal",
+  "version": "1.0.0",
+  "description": "week-03 tdd strictEqual homework",
+  "main": ".eslintrc.js",
+  "scripts": {
```

Comment:
I realize the slides have it this way but worth mentioning that we can do this with less text also as:

```
    "lint": "eslint --format codeframe",
    "test": "mocha ./tests/*.test.js"
```